src/__tests__/fixtures/WelcomeComponent.js

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue'
+
+export default Vue.component('welcome-component', {
+  render (h) {
+    return h('span', [
+      'Welcome, ',
+      this.$slots.default,
+      '!'
+    ])
```

Comment:
Could you please use the JSX syntax, as we do it for the other fixtures components?

Reply:
Wasn't sure how to do it with JSX. I tried `<slot />` but that only works when vue compiles the template, it seams. But I learned something new today, thanks :)